codelabs/getting_started/getting_started.md

```diff
@@ -134,35 +134,44 @@ A Blockly workspace has two main components:
 
 The toolbox may be organized into categories, and may contain both single blocks and groups of blocks. A well-organized toolbox helps the user to explore the available blocks and understand the capabilities of the underlying system.
 
-A toolbox is defined in XML and passed into the workspace constructor through an options struct.
+A toolbox is defined as a JavaScript object and passed into the workspace constructor through an options struct.
+
+For more information on this JSON format and toolbox configuration, including category creation, click <a href="https://developers.google.com/blockly/guides/configure/web/toolbox">here</a>.
```

Comment:
I think it is recommended to put a link around words that better describe where it is linking to. Could we update this to "including category creation, please see our <a>toolbox documentation</a> for more information."?